### ISSUE TITLE
experiments on MSRP

### ISSUE THREAD

**Issue by lidra** [taoshen58/DiSAN#13]
hi, I am very glad that you share your code, it is nice.
I try to train a model on MSRP dataset, but the result seems bad. The model works good at training set but works bad on test set. I am curious that have you tried to train model on MSRP and how about the results?

**Comment by taoshen58** [comment]
I don't know about this dataset. What's the size of the dataset?

**Comment by lidra** [comment]
MSRP(https://www.microsoft.com/en-us/download/details.aspx?id=52398) is a dataset about text similarity. And the train set is 4077, the test set is 1726. 
I don't know how to gain a good accuracy on MSRP. The accuracy is about 70, but the best results reach to 85+.

**Comment by taoshen58** [comment]
Have you tested some popular baselines, e.g., LSTM, GRU and CNNs?
